pages/examples/included-in-the-accessibility-tree.md

```diff
@@ -6,53 +6,53 @@ These are examples of the [Included in the accessibility tree][] definition. The
 
 ## Default inclusion
 
-This `span` element is included in the accessibility tree (by default, elements are included in the accessibility tree).
+This `h3` element is included in the accessibility tree (by default, elements are included in the accessibility tree).
 
 ```html
-<span>ACT rules</span>
+<h3>ACT rules</h3>
 ```
 
 ## Removed with CSS display
 
-This `span` element is not included in the accessibility tree because it is hidden to everybody by the CSS property.
+This `h3` element is not included in the accessibility tree because it is hidden to everybody by the CSS property.
 
 ```html
-<span style="display:none">ACT rules</span>
+<h3 style="display:none">ACT rules</h3>
 ```
 
 ## Removed with aria-hidden
 
 This `span` element is not included in the accessibility tree because it is explicitly removed by the `aria-hidden` attribute.
 
 ```html
-<span aria-hidden="true">ACT rules</span>
+<h3 aria-hidden="true">ACT rules</h3>
 ```
 
 ## Included but off screen
 
-This `span` element is positioned off screen, hence is not [visible][], but is nonetheless included in the accessibility tree.
+This `h3` element is positioned off screen, hence is not [visible][], but is nonetheless included in the accessibility tree.
 
 ```html
-<span style="position: absolute; top:-9999em">ACT rules</span>
+<h3 style="position: absolute; top:-9999em">ACT rules</h3>
 ```
 
 ## Removed but not ignored
 
-Although the `span` element with an `id` of "label" is not itself included in the accessibility tree, it still provides an [accessible name][] to the other `span`, via the `aria-labelledby` attribute. Thus, it is still indirectly exposed to users of assistive technologies. Removing an element from the accessibility tree is not enough to remove all accessibility concerns from it since it can still be indirectly exposed.
+Although the `h3` element with an `id` of "label" is not itself included in the accessibility tree, it still provides an [accessible name][] to the `span`, via the `aria-labelledby` attribute. Thus, it is still indirectly exposed to users of assistive technologies. Removing an element from the accessibility tree is not enough to remove all accessibility concerns from it since it can still be indirectly exposed.
```

Comment:
```suggestion
Although the `h2` element with an `id` of "label" is not itself included in the accessibility tree, it still provides an [accessible name][] to the `h3`, via the `aria-labelledby` attribute. Thus, it is still indirectly exposed to users of assistive technologies. Removing an element from the accessibility tree is not enough to remove all accessibility concerns from it since it can still be indirectly exposed.
```